code/game/mecha/combat/marauder.dm

```diff
@@ -162,78 +162,112 @@
 	else
 		icon_state = initial_icon
 
+/spell/mech/marauder
+	hud_state = "marauder"
+	override_icon = 'icons/mecha/mecha.dmi'
 
-/obj/mecha/combat/marauder/verb/toggle_thrusters()
-	set category = "Exosuit Interface"
-	set name = "Toggle thrusters"
-	set src = usr.loc
-	set popup_menu = 0
-	if(usr!=src.occupant)
+/spell/mech/marauder/thrusters
+	name = "Toggle thrusters"
+	desc = "Toggle mech thrusters."
+	hud_state = "jetpack-void"
+	override_icon = 'icons/obj/tank.dmi'
+
+/spell/mech/marauder/thrusters/cast(list/targets, mob/user)
+	if(user!=linked_mech.occupant)
 		return
-	if(src.occupant)
-		if(get_charge() > 0)
```

Comment:
is there a reason this is no longer checked?

Reply:
whoopsies

Reply:
This should be checked in `cast_check` if you want to be clean about it